### PR TITLE
fix: lodestar vc faster startup

### PR DESCRIPTION
**Pull request by barnabasbusa**

### CODE REVIEW

**Review by MetaFomos — APPROVED** [review]

**Review by MetaFomos — APPROVED** [review]

**Comment by pk910** [comment]
Tested & works fine.
However, this uses a new flag `--disableKeystoresThreadPool`, which is not recognized by older images (including the latest stable release & pectra images) yet.
If the flag is not known, lodestar errors out with `✖ Unknown argument: disableKeystoresThreadPool`, which prevents us from using these images and effectively breaks out current testing setup.
We should not merge this PR in until the flag is recognized by the latest stable release & all important feature branches (pectra, peerdas, verkle, ...)

**Comment by nflaig** [comment]
we had to cut and patch release and also included `--disableKeystoresThreadPool` flag in there, so we are good on our next and latest release but electra and peerdas branch still needs to be updated with the change

**Comment by barnabasbusa** [comment]
@nflaig any update on this? Does electra/peerdas branch expected to get this flag update anytime soon?

**Comment by nflaig** [comment]
> @nflaig any update on this? Does electra/peerdas branch expected to get this flag update anytime soon?

Some time this week, although unclear for peerdas branch, will keep you updated once those branches support the flag

File: src/package_io/input_parser.star
Comment (nflaig) [inline src/package_io/input_parser.star:39]: I guess this can be reverted since we have it now in our latest image

Comment (barnabasbusa) [reply src/package_io/input_parser.star]: Fixed.

**Comment by barnabasbusa** [comment]
Currently waiting for:

- [ ] Pectra branch (https://github.com/ChainSafe/lodestar/tree/electra-fork)
- [ ] Peerdas branch (https://github.com/ChainSafe/lodestar/tree/peerDAS)

to support this new flag before we can merge it

**Review by pk910 — APPROVED** [review]

**Pull request merged**
